### PR TITLE
Added the very first implementation of CCEditBox for Win32.

### PR DESCRIPTION
**Pull request by jozef-pridavok**
Here is the very first version of CCEditBox for Windows. Useful for testing...

### CODE REVIEW

File: extensions/GUI/CCEditBox/CCEditBoxImplWin.cpp [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+
+/*
+
+#include "CCEditBoxImplWin.h"
+#include "CCEditBox.h"
+#include "CCEGLView.h"
+
+NS_CC_EXT_BEGIN
+
+CCEditBoxImpl* __createSystemEditBox(CCEditBox* pEditBox)
```
Comment (dumganhar) [inline extensions/GUI/CCEditBox/CCEditBoxImplWin.cpp:252]: Why there are the same functionality in this file? The same as Line36.

Comment (dumganhar) [inline extensions/GUI/CCEditBox/CCEditBoxImplWin.cpp:252]: Oh, my fault, i just notice that was commented.

**Comment by jozef-pridavok** [comment]
It's an "in-place" version of CCEditBox, but there are problems when adding native controls to the OpenGL window...

**Comment by dumganhar** [comment]
Hi @jpridavok 
Did you notice that some UTF-8 letters can't be shown in CCEditBox?
I use Chinese to test it. I think other languages also have this problem.

**Comment by dumganhar** [comment]
Thank you, merged.
But could you resolve the issue i mentioned?

**Comment by jozef-pridavok** [comment]
Hello,
yes, sure, I am going to resolve it. (there is just ANSI version, that's easy to add Unicode support)... I'll commit PR...

**Comment by dumganhar** [comment]
OK, Thanks.

**Pull request merged**